sample/build.gradle

```diff
@@ -4,6 +4,7 @@ dependencies {
     compile project(":cfg4k-core")
     compile project(":cfg4k-json")
     compile project(":cfg4k-bytebuddy")
+    compile libraries.jetbrains.spek.api
```

Comment:
[`ConfigProviderTest`](https://github.com/jdiazcano/cfg4k/blob/master/sample/src/main/kotlin/com/jdiazcano/sample/ConfigProviderTest.kt) is under main source code directory, not under `test`. I would guess it is there intentionally, so here Spek API is needed as a `compile` dependency.

Reply:
I will remove that dependency. Anyways it is just the sample and I wanted to revamp that module to include more samples and more concise ones.